apps/dashboard/src/pages/community/fast-track.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, SimpleGrid, useBreakpointValue } from "@chakra-ui/react";
+import { ChakraNextImage } from "components/Image";
+import { Heading, Text } from "tw-components";
+
+const chainsIcons = [
+  {
+    id: "treasure-icon",
+    src: require("../../../public/assets/startup-program/coinbase-icon.png"),
+    alt: "chain",
+  },
+  {
+    id: "chain-icon",
+    src: require("../../../public/assets/startup-program/chain-icon1.png"),
+    alt: "chain",
+  },
+  {
+    id: "chain-layer",
+    src: require("../../../public/assets/startup-program/layer3-icon.png"),
+    alt: "chain",
+  },
+];
+
+const chainsIcons2 = [
+  {
+    id: "treasure-icon",
+    src: require("../../../public/assets/startup-program/chain-icon2.png"),
+    alt: "chain",
+  },
+  {
+    id: "chain-icon",
+    src: require("../../../public/assets/startup-program/chain-icon3.png"),
+    alt: "chain",
+  },
+  {
+    id: "chain-layer",
+    src: require("../../../public/assets/startup-program/chain-icon4.png"),
+    alt: "chain",
+  },
+];
+
+const FastTrack = () => {
+  // Check if the screen is larger than "lg"
+  const isLargeScreen = useBreakpointValue({
+    base: false,
+    sm: false,
+    md: false,
+    lg: true,
+  });
+  return (
+    <Box>
+      {!isLargeScreen && (
+        <Box>
```

Comment:
this can be done with `display={{base: "none", lg: "block"}}` instead

Reply:
this `display={{base: "none", lg: "block"}} ` is not working for me in this case, let me try it again and test properly